### PR TITLE
Fix proffered username sso configuration

### PR DESCRIPTION
**Pull request by aindriu-aiven**
I have added additional information on setting up SSO with Klaw after an issue was raised recently.

### CODE REVIEW

File: docs/authentication-authorization/authentication/google-account.md [hunk 1]
```diff
@@ -38,15 +38,20 @@ in the Klaw Core module to enable the use of Google account login.
 
     `klaw.superadmin.default.username=superadmin@company.com`
 
-6.  If you have already signed up with Google, you will be redirected to the Klaw home page.
+6.  Each SSO provider can have a different "preferred username" attribute which identifies your user correctly.
+    For Google update this attribute in the application.properties as below.
+
+    `klaw.ad.username.attribute=email`
```
Comment (muralibasani) [inline docs/authentication-authorization/authentication/google-account.md:44]: We should not force the organization to update this claim, as preferred_username can also be present in google SSO claims.

**Review by muralibasani — CHANGES_REQUESTED** [review]
We could update klaw BE code to handle this claim internally, instead of asking user to update the claim. Ideally Klaw is dealing only with preferred usernames and not with email ids. And preferred usernames can be email ids.

IMO, this change is not required, rather we should update BE code in klaw

File: docs/authentication-authorization/authentication/third-party-account.md [hunk 2]
```diff
@@ -46,7 +46,13 @@ module.
    The `imageURI` property specifies the URL of the logo image. The image must be uploaded to
    the `assets/images/clients` folder before you can use this property.
 
-7. If you have already signed up, you will be directed to the Klaw home
+7. Each SSO provider can have a different "preferred username" attribute that identifies your user correctly.
+   Check your provider's documentation and select the attribute that defines the email address.
```
Comment (muralibasani) [inline docs/authentication-authorization/authentication/third-party-account.md:50]:
```suggestion
   Check your provider's documentation and select the attribute that defines the username/login name.
```

File: docs/authentication-authorization/authentication/third-party-account.md [hunk 3]
```diff
@@ -46,7 +46,13 @@ module.
    The `imageURI` property specifies the URL of the logo image. The image must be uploaded to
    the `assets/images/clients` folder before you can use this property.
 
-7. If you have already signed up, you will be directed to the Klaw home
+7. Each SSO provider can have a different "preferred username" attribute that identifies your user correctly.
+   Check your provider's documentation and select the attribute/claim that defines the username/login name.
+   Update this attribute in the `application.properties` file, replacing 'preferred_username' with their attribute.
+
+   `klaw.ad.username.attribute=preferred_username`
```
Comment (muralibasani) [inline docs/authentication-authorization/authentication/third-party-account.md:53]: We can update this pr abit based on the pr https://github.com/Aiven-Open/klaw/pull/2556

**Comment by aindriu-aiven** [comment]
I've overridden DCO as the unsigned commits were mine.

**Review by muralibasani — APPROVED** [review]

**Pull request merged**